pkg/schedule/simple_schedule.go

```diff
@@ -1,65 +0,0 @@
-package schedule
```

Comment:
Removed package with simple schedule which equals to windowed schedule without determined start and stop timestamp, so only windowed type has been kept. Notice that this only impact how the code looks like. For users, removing that will be **unvisible** - task with simple schedule will work as so far.